### PR TITLE
Welcome page FAQ update for users/builders

### PR DESCRIPTION
**Pull request by PopDaph**
I tried to keep it simple. 

### Welcome page for users
Capture d’écran 2023-09-22 à 16 33 07

### Welcome page for builders

Capture d’écran 2023-09-22 à 16 32 46

### CODE REVIEW

File: front/pages/w/[wId]/assistant/new.tsx [hunk 1]
```diff
@@ -224,10 +229,33 @@ export default function AssistantNew({
...
                 }}
               />
-              <StartHelperConversationButton
-                content="Hey @helper, how do I use the assistant?"
-                handleSubmit={handleSubmit}
-              />
+              {isBuilder ? (
+                <>
+                  <Button
+                    variant="primary"
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/new.tsx:235]: Maybe we can do Manage assistant non primary?

File: front/pages/w/[wId]/assistant/new.tsx [hunk 2]
```diff
@@ -238,16 +266,46 @@ export default function AssistantNew({
...
+                  />
+                  <StartHelperConversationButton
+                    content="@helper, what's the level of security and privacy dust offers?"
+                    handleSubmit={handleSubmit}
+                  />
+                </div>
+              ) : (
+                <div className="flex flex-wrap gap-2">
+                  <StartHelperConversationButton
+                    content="@helper, what are the limitations of Assistants?"
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/new.tsx:296]: Let's move this one in the middle it's sad to start a FAQ with that??

File: front/pages/w/[wId]/assistant/new.tsx [hunk 3]
```diff
@@ -224,10 +229,33 @@ export default function AssistantNew({
...
+                    label="Create a new Assistant"
+                    onClick={() => {
+                      void router.push(
+                        `/w/${owner.sId}/builder/assistants/new`
+                      );
+                    }}
+                  />
+                </>
+              ) : (
+                <StartHelperConversationButton
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/new.tsx:254]: I'd be in favor of removing this one, possibly moving it to the FAQ?

File: front/pages/w/[wId]/assistant/new.tsx [hunk 4]
```diff
@@ -224,10 +229,33 @@ export default function AssistantNew({
...
+                <>
+                  <Button
+                    variant="primary"
+                    icon={Cog6ToothIcon}
+                    label="Manage Assistants"
+                    onClick={() => {
+                      void router.push(`/w/${owner.sId}/builder/assistants`);
+                    }}
+                  />
+                  <Button
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/new.tsx:242]: The three buttons layout is responsive?

Comment (PopDaph) [reply front/pages/w/[wId]/assistant/new.tsx:242]: Capture d’écran 2023-09-22 à 16 28 41
🙇🏻‍♀️

Comment (spolu) [reply front/pages/w/[wId]/assistant/new.tsx:242]: yay

**Comment by PopDaph** [comment]
@spolu feedback applied, screenshots updated

**Review by spolu — APPROVED** [review]

**Comment by spolu** [comment]
Actually incoherence assistant / Assistant betweet queswtions in FAQ

File: front/pages/w/[wId]/assistant/new.tsx [hunk 5]
```diff
@@ -168,7 +173,7 @@ export default function AssistantNew({
               </p>
               <div>
                 <StartHelperConversationButton
-                  content="Hey @helper, how do I use the assistant?"
+                  content="Hey @helper, How can I interact with an assistant?"
```
Comment (spolu) [inline front/pages/w/[wId]/assistant/new.tsx:176]: Lower case here but upper case elsewhere ?

**Review by spolu — CHANGES_REQUESTED** [review]

File: front/pages/w/[wId]/assistant/new.tsx (hunk 5 above)
Comment (spolu) [reply front/pages/w/[wId]/assistant/new.tsx:176]: Both ways are fine to me, but there should be only one way :)

**Comment by PopDaph** [comment]
@spolu fixed!

**Review by spolu — APPROVED** [review]

**Pull request merged**